VEDA/settings.py

```diff
@@ -137,6 +137,7 @@
     'rest_framework.authtoken',
     'oauth2_provider',
     'rest_framework',
+    'django_filters',
```

Comment:
**Note to reviewers:**  This is added to resolve `TemplateNotExist` error when accessing rest api views.

See https://github.com/edx/credentials/pull/284/files and carltongibson/django-filter#562